ddtrace/pin.py

```diff
@@ -7,4 +7,4 @@
     "The ddtrace.trace.Pin module is deprecated and will be removed.",
     message="Import ``Pin`` from the ddtrace.trace package.",
     category=DDTraceDeprecationWarning,
-)
+)
```

Comment:
We don't actually need to touch this file do we?

Reply:
nope